src/main/resources/view/informationView.fxml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.ScrollPane?>
+<?import javafx.scene.control.TextArea?>
+<?import javafx.scene.layout.VBox?>
+<?import javafx.scene.text.*?>
+<?import java.net.URL?>
+<ScrollPane xmlns="http://javafx.com/javafx"
+            xmlns:fx="http://javafx.com/fxml"
+            fx:id="parent"
+            prefWidth="260"
+            fx:controller="com.example.controller.InformationController">
+
+    <VBox>
+        <stylesheets>
+            <URL value="@/css/information.css"/>
+        </stylesheets>
+        <TextFlow textAlignment="CENTER" prefWidth="250">
```

Comment:
I'd adjust simulationOptionView width since it's smaller than 250 and consequently changes its size.

Reply:
Fixed by removing setManaged() from InformationController -> clearView()